packages/fx-core/tests/plugins/resource/bot/unit/index.test.ts

```diff
@@ -2,13 +2,13 @@
 // Licensed under the MIT license.
 import "mocha";
 import * as chai from "chai";
-import fs from "fs-extra";
+import fs = require("fs-extra");
 import * as sinon from "sinon";
-import { default as chaiAsPromised } from "chai-as-promised";
-import AdmZip from "adm-zip";
-import path from "path";
+//import { default as chaiAsPromised } from "chai-as-promised";
```

Comment:
Why we comment this? if it is not used, we'd better to remove it instead of comment it